libraries/AP_OSD/AP_OSD.h

```diff
@@ -99,8 +104,100 @@ class AP_OSD_AbstractScreen
     char u_icon(enum unit_type unit);
     float u_scale(enum unit_type unit, float value);
 
+    //Symbols
+    static constexpr uint8_t _symbols[78][2] = {
+        {0xB9,0x0C},    // SYM_M
+        {0xBA,0x7D},    // SYM_KM
+        {0x0F,0x0F},    // SYM_FT
+        {0xBB,0x7E},    // SYM_MI
```

Comment:
More random constants?

There are even constants for these elsewhere?

Reply:
these used to be a bunch of #defines, are you suggesting creating 2 big list of static const uint8_t SYM_*, one for regular OSD and one for DISPLAYPORT, the constexpr array would then reference those static const?